packages/graphback-datasync/src/DatasyncPlugin.ts

```diff
@@ -76,13 +77,13 @@ export class DatasyncPlugin extends GraphbackPlugin {
 
 
             // Diff queries
-            if (this.pluginConfig.generateDeltaQueries) {
-                const diffQuery = `${model.graphqlType.name}Delta`
+            if (parseMarker('delta', model.graphqlType.description)) {
```

Comment:
@craicoverflow I thought we changed api in https://github.com/aerogear/graphql-metadata

Reply:
Yes, you can now do:

`parseMarker('delta', model.graphqlType)` or preferably `parseMetadata('delta', model.graphqlType)`.

`parseMarker` accepts to make it backwards compatible.